Source/Core/DolphinWX/Config/WiiConfigPane.cpp

```diff
@@ -21,6 +21,18 @@
 #include "DolphinWX/WxUtils.h"
 #include "UICommon/USBUtils.h"
 
+// SYSCONF uses 0 for bottom and 1 for top, but showing top first in the GUI
+// makes the most sense, so we number them the other way in the GUI code
+static int TranslateSensorBarPosition(int position)
+{
+  if (position == 0)
+    return 1;
+  if (position == 1)
+    return 0;
+
+  return position;
```

Comment:
Because it's a case that shouldn't be reached, and you should usually try to enforce your invariants. That's the "why". But if it doesn't matter, then don't worry about it.